### DISCUSSION TITLE
I get email that somebody replied on an open issue but the post is not present on Github issue itself.

### DISCUSSION THREAD

**Discussion by maniacx** [community/community#110129] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

On my  repo, I have an Github issue opened # 1 with on going discussion
I have recieved an email from Github,  that a new user "abc" has posted a "xyz" on the issue # 1.
But when I go to reply on Github on issue # 1 , the post is missing or not updated in the discussion. so I cannot reply.

How long does it takes to update the the post after recieving the email? I has been 1 hour already since I recieved the email but still nothing on Github web or Github mobile app.?

If the user deleted the post, after posting, will I get an email regarding that too?

I dont know if it is a bug with Github that the post is not displayed or if the user deleted the post after replying.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by binury** [comment, marked as answer]
This question has been asked a few times. https://stackoverflow.com/questions/38986589/does-github-send-a-notification-when-an-issue-comment-is-edited

**Reply by maniacx** [reply]
So I have to just assume that the non existant post is due to user deleting the post and not due to a Github bug and there is no way of exactly knowing what could be the cause of non existant post. Great nightmare!!
